H7A3Zi_AzureRTOS_ThreadX/Core/Src/app_threadx.c

```diff
@@ -43,12 +44,23 @@
 
 /* Private variables ---------------------------------------------------------*/
 /* USER CODE BEGIN PV */
-
+UART_HandleTypeDef huart3;
+uint8_t thread_stack1[THREAD_STACK_SIZE];
```

Comment:
Is it right way to allocate memory to thread?

Reply:
No, I have allocated it using pool_allocate API of ThreadX.